src/views/blocks/tickets.php

```diff
@@ -62,67 +63,79 @@
+		novalidate
+	>
+		<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title">
+			<?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?>
+		</h2>
+		<input type="hidden" name="tribe_tickets_saving_attendees" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar_data" value="" id="tribe_tickets_block_ar_data"/>
 		<?php
-		// We're assuming that all the currency is the same here.
-		$currency_symbol     = $currency->get_currency_symbol( $ticket->ID, true );
-		$this->template( 'blocks/tickets/footer', [ 'tickets' => $tickets, 'currency_symbol' => $currency_symbol ] );
+		$this->template(
+			'components/notice',
+			[
+				'id'              => 'tribe-tickets__notice__tickets-in-cart',
+				'notice_classes'  => [
+					'tribe-tickets__notice--barred',
+					'tribe-tickets__notice--barred-left',
+				],
+
+				'content_classes' => [
+					'tribe-common-b3',
+				],
+				'content'         => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' ),
+			]
+		);
 		?>
-	<?php else : ?>
-		<?php $this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] ); ?>
-	<?php endif; ?>
-	<?php
-		ob_start();
-		/**
-		 * Allows filtering of extra classes used on the tickets-block loader
-		 *
-		 * @since  4.11.0
-		 *
-		 * @param  array $classes The array of classes that will be filtered.
-		 */
-		$loader_classes = apply_filters( 'tribe_tickets_block_loader_classes', [ 'tribe-tickets-loader__tickets-block' ] );
-		include Tribe__Tickets__Templates::get_template_hierarchy( 'components/loader.php' );
-		$html = ob_get_contents();
-		ob_end_clean();
-		echo $html;
-	?>
-</form>
-<div class="tribe-common">
-	<span id="tribe-tickets__modal_target"></span>
+
+		<?php $this->template( 'blocks/tickets/commerce/fields', [ 'provider' => $provider, 'provider_id' => $provider_id ] ); ?>
+
+		<?php if ( $has_tickets_on_sale ) : ?>
+			<?php foreach ( $tickets_on_sale as $key => $ticket ) : ?>
+				<?php $ticket_symbol = $currency->get_currency_symbol( $ticket->ID, true ); ?>
+				<?php
+					$this->template(
+						'blocks/tickets/item',
+						[
+							'ticket'          => $ticket,
+							'key'             => $key,
+							'currency_symbol' => $ticket_symbol,
+						]
+					);
+				?>
+			<?php endforeach; ?>
+			<?php
+			// We're assuming that all the currency is the same here.
+			$currency_symbol = $currency->get_currency_symbol( $ticket->ID, true );
+			$this->template(
+				'blocks/tickets/footer',
+				[
+					'tickets'         => $tickets,
+					'currency_symbol' => $currency_symbol,
+				]
+			);
+			?>
+		<?php else : ?>
+			<?php $this->template( 'blocks/tickets/item-inactive', [ 'is_sale_past' => $is_sale_past ] ); ?>
+		<?php endif; ?>
```

Comment:
just a suggestion, but you could probably do all this in one `<?php ?>` tag rather than opening and closing every line.